plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -73,3 +73,60 @@ Compressor.Threshold="Threshold (dB)"
 Compressor.AttackTime="Attack (ms)"
 Compressor.ReleaseTime="Release (ms)"
 Compressor.OutputGain="Output Gain (dB)"
+Sepia.Tone="Sepia Tone"
```

Comment:
Looking through the various language string files, I see the use of "X.Y" for multi-word strings.